source/reference/versioning.txt

```diff
@@ -82,6 +82,69 @@ released, version 2.0 would no longer receive patch releases. At the
 same time, version 1.3 would continue to receive patches until version
 1.4 was released.
 
+.. _c2c-versioning-live-upgrade:
+
+Live Upgrade
+~~~~~~~~~~~~
+
+.. versionadded:: 1.6.0
+
+Starting in ``mongosync`` 1.6.0, you can upgrade ``mongosync`` without
+restarting data synchronization operations from the beginning. This is
+known as a *live upgrade*. After the live upgrade, ``mongosync``
+continues operations that were in progress before the upgrade.
+
+You can live upgrade ``mongosync`` to the latest version.
+
+To live upgrade ``mongosync``, perform these steps:
+
+.. procedure::
+   :style: normal
+
+   .. step:: (Optional) Pause the ``mongosync`` processes
+   
+      To pause ``mongosync``, run :ref:`c2c-api-pause` on each running
+      ``mongosync`` process.
+      
+      You can restart the ``mongosync`` processes later from the paused
+      state, as shown in the final step.
+      
+      .. tip::
+
+         If you don't want to immediately resume data synchronization
+         operations after live upgrade, consider pausing the
+         ``mongosync`` processes. This is useful if you're coordinating
+         operations from multiple ``mongosync`` processes.
+
+   .. step:: Stop the ``mongosync`` processes
```

Comment:
**Suggestion**

Maybe this is a question for tech review, but can we add some detail about how to stop a process?

I am wondering if we can add a bit of content here to the stop each ``mongosync`` process. I was looking at the rest endpoints and noticed there was not an endpoint for`stop` (only pause). I dont think there is a location in the docs that mentions specifically how to stop a process so adding some tabs with the various commands for Mac / Win / Linux OSs would be helpful here.

Reply:
As this varies between different versions of Windows (each variation of Windows does this very differently), and the other OSes, this is something we assume the reader knows something about. They are assumed to know the basics of OS administration and we don't typically add those steps to our docs (and neither do other database vendors from what I've seen).